bin/crew

```diff
@@ -1711,6 +1718,8 @@ def archive_package(crew_archive_dest)
     end
   end
   system "sha256sum #{crew_archive_dest}/#{pkg_name} > #{crew_archive_dest}/#{pkg_name}.sha256"
+  # Make sure the package file has runtime dependencies added properly.
+  system "#{CREW_LIB_PATH}/tools/getrealdeps.rb #{@pkgName}"
```

Comment:
Use `require` for ruby code?

Reply:
Do you have a good way of making this happen while also allowing it to run standalone?

Reply:
Let's hold off on this for now, as this will require significant refactoring...